include/nfc/ndef/nfc_text.rst

```diff
@@ -0,0 +1,75 @@
+.. _nfc_text:
+
+Text records
+############
+
+Text records contain descriptive text and can be added to any type of message.
+This kind of record can be used in a message together with other records to provide extra information.
+The Text Records module provides functions for creating and encoding text records.
+
+Text records contain a payload (the actual text string) and a language identifier.
+Note that a message can contain only one text record per language code, so you cannot add two different text records with language code "en" to the same message, for example.
+
+The following code example shows how to generate a raw text message that contains a text record.
+
+First, generate the text record.
+In this example, the record contains the text "Hello World!" and the language code "en":
+
+.. code-block:: c
+
```

Comment:
I copied the code blocks in this file from the sample - but please check.

Reply:
I did and it looks alright. Should I remove the snippets from the sample code if they aren't used? Or are we going to introduce at some point similar snippet-like functionality that we used in the previous doc framework?

Reply:
I don't think there are code snippets like that in RST - @carlescufi ?

Reply:
@ru-fu I am not sure what you mean by "like that"? maybe you can point me to an example?

Reply:
You can include source code directly from another file if that is what you mean, take a look at how to do it here:

```
.. literalinclude:: ../../../samples/bluetooth/beacon/src/main.c
   :language: c
   :lines: 19-
   :linenos:
```

https://raw.githubusercontent.com/zephyrproject-rtos/zephyr/master/doc/guides/bluetooth/index.rst